site/content/api/Event-API.md

```diff
@@ -31,8 +31,15 @@ Hono supports *AT LEAST ONCE* delivery of *Event* messages only. A client theref
 
 **Message Flow**
 
+The following sequence diagram illustrates the flow of messages involved in the *MQTT Adapter* uploading an event data message to Hono Messaging's Event endpoint.
+
 ![Upload event data flow](../uploadEvent_Success.png)
 
+1. *MQTT Adapter* sends event data for device `4711`.
+   1. *Hono Messaging* successfully verifies that device `4711` of `TENANT` exists and is enabled by means of validating the *registration assertion* included in the message (see [Device Registration]({{< relref "api/Device-Registration-API.md#assert-device-registration" >}})) and forwards data to *AMQP 1.0 Messaging Network*.
+1. *AMQP 1.0 Messaging Network* acknowledges reception of the message.
+   1. *Hono Messaging* acknowledges reception of the message. In contrast to telemetry data, the protocol adapter waits explicitly for the accepted disposition before it signals the successful delivery to the device.
```

Comment:
we need to be careful here with the term *delivery* because people might erorneously believe that it is about delivering the event to a consuming application (which it is not). So we should probably better write something like:

In contrast to telemetry data, the protocol adapter awaits the disposition from the *AMQP 1.0 Messaging Network* 
 before signaling to the device that it has accepted the message for processing.

Reply:
:+1: that is better, I take exactly this into the description.